libraries/botbuilder-core/tests/telemetryMiddleware.test.js

```diff
@@ -102,22 +169,18 @@ describe(`TelemetryMiddleware`, function () {
             .assertReply('echo:foo')
             .send('bar')
             .assertReply(activity => assert.equal(activity.type, ActivityTypes.Typing))
-            .assertReply('echo:bar')
-            .then(done);
```

Comment:
as part of the `try/catch` change above, I moved all of the `done()` calls into the `try/catch` wrapper